classes/person.rb

```diff
@@ -3,7 +3,7 @@
 require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :title, :author, :rental
```

Comment:
👍 Good job on the Person class! However, there's a small typo that needs attention.  

- [ ] In your `attr_accessor`, you wrote `:rental`, but it should be `:rentals` to match the instance variable. Please consider updating it for consistency. Thanks for your efforts! Addressing this typo will ensure the correct functionality of your code. Keep up the great work! 👏🚀